common/src/test/java/com/tencent/rss/common/util/RssUtilsTest.java

```diff
@@ -121,4 +123,69 @@ public void testShuffleIndexSegment() {
     assertEquals(6, shuffleDataSegments.get(2).getLength());
     assertEquals(1, shuffleDataSegments.get(2).getBufferSegments().size());
   }
+
+  @Test
+  public void testLoadExtentions() {
+    try {
+      List<String> exts = Arrays.asList("Dummy");
+      try {
+        RssUtils.loadExtensions(RssUtilTestDummy.class, exts, 1);
+      } catch (RuntimeException e) {
+        assertTrue(e.getMessage().startsWith("java.lang.ClassNotFoundException: Dummy"));
+      }
+      exts = Arrays.asList("com.tencent.rss.common.util.RssUtilsTest$RssUtilTestDummyFailNotSub");
+      try {
+        RssUtils.loadExtensions(RssUtilTestDummy.class, exts, 1);
+      } catch (RuntimeException e) {
+        assertTrue(e.getMessage().contains("RssUtilTestDummyFailNotSub is not subclass of "
+            + "com.tencent.rss.common.util.RssUtilsTest$RssUtilTestDummy"));
+      }
+      exts = Arrays.asList("com.tencent.rss.common.util.RssUtilsTest$RssUtilTestDummyNoConstructor");
+      try {
+        RssUtils.loadExtensions(RssUtilTestDummy.class, exts, "Test");
+      } catch (RuntimeException e) {
+        assertTrue(e.getMessage().contains("RssUtilTestDummyNoConstructor.<init>()"));
+      }
+      exts = Arrays.asList("com.tencent.rss.common.util.RssUtilsTest$RssUtilTestDummySuccess");
+      String testStr = String.valueOf(new Random().nextInt());
+      List<RssUtilTestDummy> extsObjs = RssUtils.loadExtensions(RssUtilTestDummy.class, exts, testStr);
+      assertEquals(1, extsObjs.size());
+      assertEquals(testStr, extsObjs.get(0).get());
+    } catch (Exception e) {
```

Comment:
catch is unnecessary, test failed if there has an exception

Reply:
Assert.fail is added already.

Reply:
removed